### DISCUSSION TITLE
Seeking support with SVG-based NFT preview issue

### DISCUSSION THREAD

**Discussion by i004909** [hashgraph/hedera-mirror-node-explorer#1478] [Q&A]
Hi, I'm having an issue with NFT preview. Image is type SVG. In the SVG, when using width/height at fixed pixel value (e.g. 600 x 500 px) the preview displays, and on click also opens full screen. When using 100% x 100%, which makes the SVG 'elastic', resizing itself automatically as browser window size is changed, the preview no longer appears. On clicking the blank preview box the full screen opens and SVG displays perfectly. In Hashpack, the preview works fine when using %age for the SVG size. My query - is there a specific size or ratio to use that might fix this? It's not a showstopper - but improves the quality of the NFT immensely if immediately displayed on initial screen / preview. Tnx.

**Comment by ericleponner** [comment]
Good question. Let us check and come back. Would you agree to share your svg image for us to test ?

**Reply by i004909** [reply]
Yes absolutely, appreciate your reply very much!
Herewith two test NFTs I generated. 
Dynamic SVG NFT - not previewing
https://hashscan.io/testnet/token/0.0.5071686/7
Static size SVG NFT - previews OK.
https://hashscan.io/testnet/token/0.0.5071686/15
If you open each by clicking on the preview, then resize the browser window a bit you'll see the desired "elastic" behavior in /15. Once image is open just right click, view source to see the full SVG text. 
We're just trying to figure out if there's a specific size of the SVG or its viewbox which might make the preview work... we've tried using preserveAspectRatio="xMidYMid meet" and seems not to help. Happy to progress our project with nft  /15. but would ideally like to use /7 as our template. 
The script in the SVG is being moved to server side as I don't think it's being executed in all cases due to permissions. We're controlling font size and word-break behavior with it. FYI - both render OK in HashPack fwiw

**Comment by svienot** [comment]
Hi - we have found a bug related to the rendering of SVG images in the NFT preview and have just pushed the fix in production.
Could you check whether this addresses adequately your issue ?

**Reply by i004909** [reply]
Thank you, will do - apologies for the delay. Catching up with dev team in the next few days then will revert. Very much appreciate your help.
